CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,149 @@
+
+Moderators are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Moderators have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+Moderators are held to a higher standard than other community members. If a
+moderator creates an inappropriate situation, they should expect less leeway
+than others.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include official IRC channels (#testing
+on irc.w3.org); GitHub repositories under the web-platform-tests organization;
+and the public-test-infra@w3.org mailing list.
+
+There may arise situations where both the WPT code of conduct and that of
+another organization (such as the WHATWG or W3C) may apply.
+For example, a WPT-focused meeting at
+[TPAC](https://www.w3.org/2002/09/TPOverview.html) would involve both the WPT
+code of conduct and the [W3C code of ethics and professional
+conduct](https://www.w3.org/Consortium/cepc/).
+In such situations we operate under all code of conducts involved.
+If you are placed in a situation where you feel this is inappropriate (e.g. if
+you believe the code of conducts involved contradict one another), please
+contact a [moderator](#moderators).
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the [moderators](#moderators).
+All complaints will be reviewed and investigated promptly and fairly.
+
+All moderators are obligated to respect the privacy and security of the
+reporter of any incident.
+
+Moderators will recuse themselves if they are directly involved in a report of
+a code of conduct violation.
+
+## Enforcement Guidelines
+
+Moderators will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct.
+These are not a consecutive set of steps; a single incident may be sufficient
+enough to proceed straight to a temporary ban, for example.
+
+### 1. Correction
+
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
+
+**Consequence**: A private, written warning from the moderators, providing
+clarity around the nature of the violation and an explanation of why the
+behavior was inappropriate. A public apology may be requested.
+
+### 2. Warning
+
+**Community Impact**: A violation through a single incident or series
+of actions.
+
+**Consequence**: A warning with consequences for continued behavior. No
```

Comment:
> No interaction with the people involved, including unsolicited interaction with those enforcing the Code of Conduct, for a specified period of time.

I'm not sure about this. How would avoiding interaction with the people involved be enforced in the first place? I feel the only way that could happen is by having the target of harassment report if it happens, which puts the weight of enforcement on them, likely after they came forward and reported an instance of inappropriate conduct in the first place.

I would remove this bit and merge it with 1. Correction, something like

```markdown
### 1. Warning

**Community Impact**: A violation through a single incident or series
of actions.

**Consequence**: A warning with consequences for continued behavior, providing
clarity around the nature of the violation and an explanation of why the
behavior was inappropriate. A public apology may be requested.
```

Reply:
Done.